library-valcon-internship/src/components/Modals/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import { PropsWithChildren } from 'react'
+
+import ReactDOM from 'react-dom'
+
+interface ModalProps {
+  show: boolean,
+  confirm: () => void
+  closeModal: () => void
+}
+
+const Modal = ({ children, show, closeModal, confirm }: PropsWithChildren<ModalProps>) => {
+  return ReactDOM.createPortal(
+    <div style={!show ? { visibility: 'hidden' } : {}} className='modal'>
```

Comment:
If you use visibility hidden, element will still be present at the page. You should manage show / hide modal at the parent component and render it or not based on the condition